backend/models/Product.js

```diff
@@ -6,6 +6,11 @@ const productSchema = new Schema(
       type: String,
       required: true,
     },
+    image: {
+      // a url or a path to an image.
```

Comment:
Remove comment

Reply:
fine

Reply:
Also comment is wrong, it is not a path, can only be a url that is leading to an actual image.